src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
```

Comment:
Try, if is possible, abstract more. For better ready and manutention. For example:

const byMotherNameAndSex = people.filter(p => p.sex === 'm'
  && people.some(m => m.name === p.mother));

  const byMotherName = people.filter(p => people.some(m => m.name
    === p.mother));

  const children = onlyWithSon
    ? byMotherNameAndSex
    : byMotherName;